admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1376,6 +1376,40 @@ $product-page-padding-bottom: 80px !default;
   > .form-group:last-child {
     margin-bottom: 0;
   }
+
+  .combination-images-selector {
```

Comment:
I know it's still WIP but I had a quick look It seems mostly good but I'm wondering why we need this extra CSS?
Can't we use the already existing CSS from the combination form?